scripts/bert/finetune_classifier.py

```diff
@@ -412,11 +413,11 @@ def preprocess_data(tokenizer, task, batch_size, dev_batch_size, max_len, vocab)
         loader_test_list.append((segment, loader_test))
     return loader_train, loader_dev_list, loader_test_list, len(data_train)
 
-
-# Get the loader.
-logging.info('processing dataset...')
-train_data, dev_data_list, test_data_list, num_train_examples = preprocess_data(
-    bert_tokenizer, task, batch_size, dev_batch_size, args.max_len, vocabulary)
+if __name__ == '__main__':
```

Comment:
would you mind highlighting the main changes made to this script? Is it mostly changing the indentation?

Reply:
Yes, deploy.py imports this script, and there was a conflict in arguments when calling it.
I moved the arguments and the parser within `if __name__ == '__main__':` for solving the conflict